problems/2.functions.js

```diff
@@ -21,16 +21,20 @@ function greeting() {
 * @modifies {global.globalCount}
 */
 function incrementGlobalCount() {
-
+  let val = global.globalCount
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val